spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/AbstractMessageListenerContainer.java

```diff
@@ -872,6 +909,12 @@ protected Exception wrapToListenerExecutionFailedExceptionIfNeeded(Exception e,
 		return e;
 	}
 
+	public interface ContainerDelegate {
```

Comment:
`@FunctionalInterface`? :smile: 
Although who uses it outside of the container...
Maybe remove that `public` modifier? Or even make it `protected` ...